diff --git a/node_modules/node-pty/binding.gyp b/node_modules/node-pty/binding.gyp
index 9e8780e..ffae5d7 100644
--- a/node_modules/node-pty/binding.gyp
+++ b/node_modules/node-pty/binding.gyp
@@ -5,9 +5,6 @@
     ],
     'conditions': [
       ['OS=="win"', {
-        'msvs_configuration_attributes': {
-          'SpectreMitigation': 'Spectre'
-        },
         'msvs_settings': {
           'VCCLCompilerTool': {
             'AdditionalOptions': [
